examples/modular-drivers/README.md

```diff
@@ -35,7 +35,7 @@ index 79fbc5c..949b5ab 100644
  smol = "1.2"
  futures-lite = "1.12.0"
  micro-rdk = {version = "0.0.3", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32"]}
-+micro-rdk-modular-driver-example = { git = "https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers" }
```

Comment:
does it need the `package` specification? I figured the workspace would direct it if the name is the same as the intended package

Reply:
it doesn't but I think it's better communication to a reader about how it actually works to include it